unstructured/partition/ocr.py

```diff
@@ -533,7 +533,9 @@ def parse_ocr_data_tesseract(ocr_data: pd.DataFrame, zoom: float = 1) -> List[Te
         text = idtx.text
         if not text:
             continue
-        cleaned_text = text.strip()
+
+        cleaned_text = str(text) if not isinstance(text, str) else text.strip()
```

Comment:
```suggestion
        cleaned_text = str(text).strip()
```
?

Reply:
@cragwolfe that works. I was just thinking it wasn't necessary to call both `str()` and `.strip()` because if it's text, all we need to do is call `.strip()` and if it's a float/int all we need to call is `str()`